strings/strings.json

```diff
@@ -1359,17 +1359,17 @@
   "achieve_stats_portals": "Demonic Invasions Stopped:",
   "achieve_stats_current_game": "Current Game",
   "achieve_perks": "Perks",
-  "achieve_perks_blackhole": "Most storage increased by %0%",
+  "achieve_perks_blackhole": "Most storage is increased by %0%",
   "achieve_perks_mass_extinction": "Gain %0x more Genes from mutations",
-  "achieve_perks_creator": "Can evolve into a race of your choice on MAD resets",
+  "achieve_perks_creator": "You can evolve into a race of your choice on MAD resets",
   "achieve_perks_creator2": "Gain a RNA and DNA starting bonus during evolution",
-  "achieve_perks_explorer": "Gained %0 free extra planet options during bioseeding",
-  "achieve_perks_enlightened": "Gained a bonus to starting knowledge capacity and gain",
-  "achieve_perks_heavyweight": "Transport fuel costs reduced by %0%",
-  "achieve_perks_dissipated1": "Coal power plants genereate +%0kW energy",
-  "achieve_perks_dissipated2": "Oil power plants genereate +%0kW energy",
-  "achieve_perks_dissipated3": "Casinos required %0kW less energy",
-  "achieve_perks_dissipated4": "Mass drivers required %0kW less energy",
+  "achieve_perks_explorer": "Gain %0 free extra planet options during bioseeding",
+  "achieve_perks_enlightened": "Gain +1 base knowledge generation and +900 base knowledge storage",
+  "achieve_perks_heavyweight": "Oil and Helium costs reduced by %0%",
```

Comment:
This change makes it misleading. It's stated as "fuel costs" extremely purposefully.